### DISCUSSION TITLE
KMS Cloud Formation template is not accessible

### DISCUSSION THREAD

**Discussion by lahirudx** [Kurento/kurento#21] [Community]
I tried to install KMS-Coturn with AWS Cloud Formation template according to docs. But the cloud formation wizard says,

"_Cannot open this file because of an error.: https://s3-eu-west-1.amazonaws.com/aws.kurento.org/KMS-Coturn-cfn-7.0.0.yaml must reference a valid S3 object to which you have access._"

Is it not publicly accessible?

**Comment by j1elo** [comment]
Converted into a bug here: https://github.com/Kurento/kurento/issues/27
Seems that the CI job that was supposed to upload the updated file didn't run properly.

**Comment by thedeec1** [comment]
Has this been resolved?  I am trying to follow the AWS Cloudformation instructions and getting the same error for version 7.1.0

https://s3-eu-west-1.amazonaws.com/aws.kurento.org/KMS-Coturn-cfn-7.1.0.yaml

An error occurred while attempting to retrieve the template. This could be due to the template not existing or insufficient permissions for the S3 bucket. Verify the template's existence and your permissions.

**Reply by slabajo** [reply]
We have been moving infrastructure for all oour jurento server building and maintenance. It seems that precisely that content has an issue. Allow us some time to fix it

**Reply by ben-carrasco** [reply]
I see that there are no updates on this--would it be possible for you to provide a direct link to the file? (maybe via a Gist or similar resource).
